### ISSUE TITLE
Fabulously optimized game exited with code 1

### ISSUE THREAD

**Issue by AzonexShits** [PojavLauncherTeam/PojavLauncher#5775]
### Describe the bug

I downloaded Fabulously optimized on the create profile then modpack profile I tried both modrinth and forge I saw a year old vid with someone trying the modpack so I'm confused since maybe it's just sodium or sum is there any modpack alternative for pojav that's updated?
[latestlog.txt](https://github.com/user-attachments/files/16157650/latestlog.txt)

### The log file and images/videos

[latestlog.txt](https://github.com/user-attachments/files/16157654/latestlog.txt)

### Steps To Reproduce

```markdown
1. Start Pojav 
2. Install fabulously optimize on modpack profile
3. Download and start the loader
```

### Expected Behavior

Work normally and run Minecraft smoothly

### Platform

```markdown
- Device model: Samsung Galaxy a23 5G
- CPU architecture:ZX ARM CORTEX-A78 @ 2.21 GHZ. 6X ARM CORTEX-A55 @ 1.80 GHZ idk that's what cpu z says
- Android version: Android 14
- PojavLauncher version: FOXGLOVE-ZO240621-332DE88-V3_OPENJDK
```

### Anything else?

I deleted all app data and re downloaded it on playstore yet it still happened

**Comment by GaucovyAero** [comment]
```
Installed version: 3.3.1 SNAPSHOT
Required version: 3.3.3

You must change the LWJGL version in your launcher to continue. This is usually controlled by the settings for a profile or instance in your launcher.
```
looks like some mod does not like the old lwjgl pojav is using, figure out what mod it is and report back

**Comment by AzonexShits** [comment]
> ```
> Installed version: 3.3.1 SNAPSHOT
> Required version: 3.3.3
> 
> You must change the LWJGL version in your launcher to continue. This is usually controlled by the settings for a profile or instance in your launcher.
> ```
> 
> looks like some mod does not like the old lwjgl pojav is using, figure out what mod it is and report back

How do I do that I don't know no shit about this  what's LWJGL?

**Comment by GaucovyAero** [comment]
lwjgl is the library that minecraft needs in order to run, one or more of your mods use method(s) which are not implemented in lwjgl 3.3.1, hence why they require 3.3.3

**Comment by AzonexShits** [comment]
> lwjgl is the library that minecraft needs in order to run, one or more of your mods use method(s) which are not implemented in lwjgl 3.3.1, hence why they require 3.3.3

So how should I find it?

**Comment by AzonexShits** [comment]
> lwjgl is the library that minecraft needs in order to run, one or more of your mods use method(s) which are not implemented in lwjgl 3.3.1, hence why they require 3.3.3
Hey I'm getting desperate right now if I replaced all the similar mods of fabulously with haste would it work? Haste works fine but I would like to run fabulously optimized I'm guessing something else is needed like config or sum I'm just asking

**Comment by apricoffee** [comment]
Sodium is the problem here. Any way to fix it?

**Comment by AzonexShits** [comment]
> Sodium is the problem here. Any way to fix it?

Idk haste has sodium so maybe I can copy the files of sodium on haste no? And additive also had sodium and it did launch but after loading into the world it crashed

**Comment by GaucovyAero** [comment]
yeah you gotta test this out yourself, sodium up to 0.4.10 can be patched but only haste has 0.5.x patched, so maybe try pulling it from there, but cant really help much more than that, if you have any other questions just ask or create another issue
